### PR TITLE
Fix some warnings

### PR DESCRIPTION
**Pull request by iamthesenate1**
I was not sure if I should delete the unused variables so I just commented them out. Please let me know if I should have done otherwise.

### CODE REVIEW

**Comment by tsujan** [comment]
@iamthesenate1 
Thanks for this! Qps needs lots of cleanup.

I might review it later -- currently very busy with other jobs.

**Comment by iamthesenate1** [comment]
Ok. There is no need to hurry.

File: src/qps.cpp [hunk 1]
```diff
@@ -1009,7 +1009,7 @@ void Qps::make_command_menu()
     }
     connect(m_command, SIGNAL(triggered(QAction *)),
             SLOT(run_command(QAction *)));
-
+*/
```
Comment (tsujan) [inline src/qps.cpp:1012]: Please don't comment out these for now.

Qps is full of strange code lines. The "Command" menu is one of them. Although it only has the Quit action, we might be able to read the mind of the original author at some point. Commenting out lines like these will make the job harder later.

Comment (iamthesenate1) [reply src/qps.cpp]: Ok I have just commented out line 992, whith a message to explain why.

File: src/qps.cpp [hunk 2]
```diff
@@ -989,7 +989,7 @@ void Qps::menu_edit_cmd()
 void Qps::make_command_menu()
 {
     // should clear SIGNAL!!!
-    QAction *act;
+    //QAction *act; - commented out to reduce warnigs when compiled
```
Comment (tsujan) [inline src/qps.cpp:992]: I'm afraid this results in a compilation failure.

Please leave it as it was.

File: src/qps.cpp [hunk 3]
```diff
@@ -1009,7 +1009,7 @@ void Qps::make_command_menu()
     }
     connect(m_command, SIGNAL(triggered(QAction *)),
             SLOT(run_command(QAction *)));
-
+
```
Comment (tsujan) [inline src/qps.cpp:1012]: Please use a setting in your text editor so that trailing spaces are removed (also in your future patches).

Comment (iamthesenate1) [reply src/qps.cpp]: Ok.

File: src/qps.cpp (hunk 2 above)
Comment (iamthesenate1) [reply src/qps.cpp:992]: Ok

**Review by tsujan — APPROVED** [review]
Thanks!

**Pull request merged**